mypy/checkexpr.py

```diff
@@ -1061,15 +1061,51 @@ def check_op(self, method: str, base_type: Type, arg: Node,
             # method of the other operand instead.
             rmethod = self.get_reverse_op_method(method)
             arg_type = self.accept(arg)
+            base_arg_node = TempNode(base_type)
+            user_error = None  # type: Tuple[Tuple[Type, Type], MessageBuilder]
```

Comment:
The type should be `Optional[Tuple[...]]`? (Not that it matters to mypy right now, but soon it hopefully will matter.)